src/app/page.tsx

```diff
@@ -1,10 +1,9 @@
 "use client";
 
 import { Navbar } from "@/components/Navbar";
-import { Chart } from "@/components/Chart";
 import MarketTable from "../components/MarketTable/index";
 import { CoinSelectorCarousel } from "../components/CoinSelectorCarousel/index";
-import { CoinPortfolioSwitch } from "@/components/CoinPortfolioSwitch";
+import { CoinCharts } from "@/components/CoinCharts";
```

Comment:
move it be fore the relative imports

Reply:
imports priority:
node_modules 
absolute imports made by you
relative imports